nautobot/docs/administration/nautobot-server.md

```diff
@@ -0,0 +1,385 @@
+
+## Getting Help
+
+To see all available management commands:
+
+```no-highlight
+$ nautobot-server help
+```
+
+All management commands have a `-h/--help` flag to list all available arguments for that command, for example:
+
+```no-highlight
+$ nautobot-server migrate --help
+```
+
+## Available Commands
+
+### `collectstatic`
+
+`nautobot-server collectstatic`
+
+Collect static files into [`STATIC_ROOT`](../../configuration/optional-settings/#static_root).
+
+```no-highlight
+$ nautobot-server collectstatic
+
+965 static files copied to '/opt/nautobot/static'.
+```
+
+!!! note
+    This is a built-in Django command. Please see the [official documentation on `collectstatic`](https://docs.djangoproject.com/en/stable/ref/django-admin/#collectstatic) for more information.
+
+### `createsuperuser`
+
+`nautobot-server createsuperuser`
+
+Creates a superuser account that has all permissions. 
+
+```no-highlight
+$ nautobot-server createsuperuser
+Username (leave blank to use 'jathan'): example
+Email address: example@localhost
+Password:
+Password (again):
+Superuser created successfully.
+```
+
+!!! note
+    This is a built-in Django command. Please see the [official documentation on `createsuperuser`](https://docs.djangoproject.com/en/stable/ref/django-admin/#createsuperuser) for more information.
+
+### `dbshell`
+
+`nautobot-server dbshell`
+
+A shell for your PostgreSQL database. This can be very useful in troubleshooting raw database issues.
+
+!!! danger
+    This is an advanced feature that gives you direct access to run raw SQL queries. Use this very cautiously as you can cause irreparable damage to your Nautobot installation.
+
+```no-highlight
+$ nautobot-server dbshell
+psql (12.6 (Ubuntu 12.6-0ubuntu0.20.04.1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+nautobot=> \conninfo
+You are connected to database "nautobot" as user "nautobot" on host "localhost" (address "::1") at port "5432".
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+nautobot=> \q
+```
+
+!!! note
+    This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
+
+### `generate_secret_key`
+
+`nautobot-server generate_secret_key`
+
+Generates a new [`SECRET_KEY`](../../configuration/required-settings/#secret_key) that can be used in your `nautobot_config.py`:
+
+```no-highlight
+$ nautobot-server generate_secret_key
+e!j=vrlhz-!wl8p_3+q5s5cph29nzj$xm81eap-!&n!_9^du09
+```
+
+### `init`
+
+`nautobot-server init [config_path]`
+
+Generates a new configuration with all of the default settings provided for you, and will also generate a unique[`SECRET_KEY`](../../configuration/required-settings/#secret_key).
+
+By default the file will be created at `$HOME/.nautobot/nautobot_config.py`:
```

Comment:
Should we have any mention of `$NAUTOBOT_ROOT` here?

Reply:
I considered it, but the configuration guide goes into a lot more detail. We could duplicate it here if you think it would make more sense?

Reply:
Given the issues we've already seen around the importer documentation being duplicated in multiple places, probably best not to duplicate that info here. Thanks.

Reply:
Yeah, that's why I attempted to address that by linking to the configuration guide just below this sentence.